mpp-ios/ContentView.swift

```diff
@@ -7,16 +7,75 @@
 
 import SwiftUI
 
+struct Station: Identifiable, Hashable {
+    let name: String
+    let id: String
+}
+
 struct ContentView: View {
+    @State private var showingAlert = false
+    
+    var stations = [
+        Station(name: "London Kings Cross", id: "KGX"),
+        Station(name: "Edinburgh Waverley", id: "EDB"),
+        Station(name: "Newcastle", id: "NCL"),
+        Station(name: "York", id: "YRK"),
+        Station(name: "Leeds", id: "LDS")
+    ]
+    @State private var departStation: Station? = nil
+    @State private var arrivalStation: Station? = nil
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundStyle(.tint)
-            Text("Hello, world!")
+            Text("Live Train Times")
+                .font(.title)
+                .fontWeight(.bold)
+                .padding(.all)
+            
+            HStack {
+                Picker("Select departure station", selection: $departStation) {
+                    Text("Select departure...").tag(Optional<Station>(nil))
+                    ForEach(stations) { station in
+                        Text(station.name).tag(station as Station?)
+                    }
+                }
+                .frame(width: 150)
```

Comment:
Not sure we should be using hardcoded sizes here, can you make it adaptive instead?
e.g. `.fixedSize(horizontal: false, vertical: true)`

Reply:
We aren't hardcoding sizes anymore, but we're trying to keep the width fixed as one half of the screen, so this is what we've got now: `.frame(
                    minWidth: 0,
                    maxWidth: .infinity
                )` is this okay?

Reply:
Looks better for sure!